microsoft/react-native-macos
# Fixed Voiceover reading of Text

#### Please select one of the following

## Summary

The contents of TextViews were not being announced by Voiceover. I.e. when Voiceover's focus was on some text element it would just read out "text" rather than whatever actual content was in the element. This is unexpected behavior and would have caused visually impaired users to be unable to read the vast majority of the textual information in our experiences.

## Changelog

[MacOS] [Fixed] - Fixed Voiceover reading of Text 

## Test Plan

1. Compile and run app.
2. Navigate to Text page
3. Turn on Voiceover.
4. Navigate to any Text.
- You should hear/see voiceover announce the text inside the element.


###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-macos/pull/476)
